src/Tools/ProcessWatchdog/Options.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using CommandLine;
```

Comment:
Using the CommandLineParser NuGet package that Michael and I have used in several other projects. Saves all that pesky hand-parsing.